runtime/js/12_io.js

```diff
@@ -102,7 +102,7 @@
       return 0;
     }
 
-    const nread = await core.opAsync("op_read_async", rid, buffer);
+    const nread = await core.read(rid, buffer);
 
     return nread === 0 ? null : nread;
   }
```

Comment:
I believe this two APIs should be completely removed in favor of `core.read` and `core.write`

Reply:
There's some other rewiring of read/write/shutdown exports to do, I would rather do that in a scoped follow-up PR